README.md

```diff
@@ -130,8 +130,8 @@ The New Relic Ruby agent also uses source code from third-party libraries. Full
 
 ## Thank You
 
-Thank you, and may your application scale to infinity plus one.
+Thank you,
 
-Lew Cirne, Founder
+New Relic Ruby agent team
 
 New Relic, Inc.
```

Comment:
What are your thoughts about removing `New Relic, Inc.` here? I'm not sure what is consistent with the other agent READMEs, but this seems redundant given line 135.

Reply:
I agree with that Kayla! I'll go ahead and make that change.